frontend/README.md

```diff
@@ -11,6 +11,7 @@ To communicate with the webapp, the user should install [Temple wallet](https://
 
 
 ```bash
+make typescript # To generated types needed by the frontend
```

Comment:
since we are suggesting to run `make` here, why not just suggest to run `make frontend`?